frontend/www/js/omegaup/components/user/ManageApiTokens.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <div>
+    <form class="form add-identity-form" @submit.prevent="handleCreateApiToken">
```

Comment:
Esta clase `add-identity-form` indica que es para un formulario donde se agregan identidades.

Si acá no tiene sentido agregar una clase extra, entonces se la puedes quitar:

```suggestion
    <form class="form" @submit.prevent="handleCreateApiToken">
```

Reply:
Listo, pensaba que (al basarme en el template de identities) usaba algún estilo que era reelevante, pero al parecer es meramente para identificar al div.